pom.xml

```diff
@@ -1315,16 +1362,79 @@
             <name>Dummy to avoid accidental deploys</name>
             <url />
         </repository>
+        <site>
+            <id>default</id>
+            <name>Mina SSHD</name>
+            <url>file://${project.build.directory}/deploy</url>
+        </site>
     </distributionManagement>
 
     <reporting>
-       <plugins>
-           <plugin>    <!-- used by PMD to create a source XREF -->
-               <groupId>org.apache.maven.plugins</groupId>
-               <artifactId>maven-jxr-plugin</artifactId>
-               <version>3.0.0</version>
-           </plugin>
-       </plugins>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jxr-plugin</artifactId>
+                <version>3.0.0</version>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <reportSets>
+                    <reportSet>
+                        <reports>
+                            <report>report</report>
+                        </reports>
+                    </reportSet>
+                    <reportSet>
+                        <id>aggregate</id>
+                        <inherited>false</inherited>
+                        <reports>
+                            <report>report-aggregate</report>
+                        </reports>
+                    </reportSet>
+                </reportSets>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-project-info-reports-plugin</artifactId>
+                <version>2.6</version>
+                <reportSets>
+                    <reportSet>
+                        <reports>
+                            <report>index</report>
+                        </reports>
+                    </reportSet>
+                </reportSets>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>2.9</version>
+                <configuration>
```

Comment:
- We already have a definition for the plugin (and with a more advanced version)
- I don't want the plugin to run as part of the normal build cycle
- I am not sure we want the tests javadoc
- The `javase/8` is hardcoded - should be derived from one of the existing properties (e.g., `javac.source` or similar)